apps/web/src/components/pages/team/team-grid.tsx

```diff
@@ -1,8 +1,9 @@
 import { cn } from '@editor/utils/cn'
```

Comment:
Unrelated to this PR probably, but this looks like a wrong import. It's importing `cn` from the editor, even though we have one in web.